### PR TITLE
fixed hasClassName in case className is not a string and has toString method.

### PR DESCRIPTION
**Pull request by f0rmat1k**
Closes #518

### CODE REVIEW

File: src/ShallowTraversal.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ export function childrenOfNode(node) {
 
 export function hasClassName(node, className) {
   let classes = propsOfNode(node).className || '';
+  classes = String(classes);
   classes = classes.replace(/\s/g, ' ');
```
Comment (ljharb) [inline src/ShallowTraversal.js]: `String(classes).replace` would be a bit more succinct, fwiw.

**Comment by f0rmat1k** [comment]
@ljharb agreed. fixed.

**Comment by ljharb** [comment]
LGTM

**Comment by blainekasten** [comment]
This branch needs to be caught up now.

**Comment by f0rmat1k** [comment]
@blainekasten using merge is ok?

**Comment by Aweary** [comment]
@f0rmat1k looks good, thanks for the PR!

**Pull request merged**